src/data/files/file.model.ts

```diff
@@ -17,7 +17,11 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import mongoose from 'mongoose';
+import mongoose, { PaginateModel } from 'mongoose';
+import { File } from 'winston/lib/winston/transports';
+import { FILE_STREAM_LIMIT } from '../../config';
+
+const mongoosePaginate = require('mongoose-paginate-v2');
 const AutoIncrement = require('mongoose-sequence')(mongoose);
```

Comment:
Yes, this is a `require` statement not import, this has been that way for a while and so I didn't update... If you import it instead of require it then it recognizes the types from the @types file and they don't match this usage... so we're stuck with this to keep the TS compiler happy. Ideal solution is to add comments here explaining why this is happening, and then to find a library (or fix this one) so that the type def matches the usage.